libcontainerd/client_linux.go

```diff
@@ -466,11 +466,18 @@ func (clnt *client) Restore(containerID string, attachStdio StdioCallback, optio
 	// events will do is change the state of the container to be
 	// exactly the same.
 	cont, err := clnt.getContainerdContainer(containerID)
-	// Get its last event
-	ev, eerr := clnt.getContainerLastEvent(containerID)
-	if err != nil || containerd_runtime_types.State(cont.Status) == containerd_runtime_types.Stopped {
-		if err != nil {
-			logrus.Warnf("libcontainerd: failed to retrieve container %s state: %v", containerID, err)
+	if err != nil {
+		logrus.Warnf("libcontainerd: failed to retrieve container %s state: %v", containerID, err)
+		return nil
```

Comment:
The old code doesn't necessarily return right away in this case. Is the change correct?